### DISCUSSION TITLE
Bootstrap Dropdown is not working in vue components

### DISCUSSION THREAD

**Discussion by bidyut18** [twbs/bootstrap#32519] [Q&A] [answered]
I'm building a vue app using bootstrap beta version. "@popperjs/core": "^2.6.0",
    "bootstrap": "^5.0.0-beta1".
The dropdown is not working in the Vue components.  Is there any solution possible. 
Here is the screenshot.
The dropdown doesn't show any visual response after clicking. dropdown menu is not showing.
![Screenshot 2020-12-17 205939](https://user-images.githubusercontent.com/69600275/102508379-52f05700-40ab-11eb-9456-a47025933ab7.jpg)
![bootstrap dropdown](https://user-images.githubusercontent.com/69600275/102508427-613e7300-40ab-11eb-8002-4ecb93d7a61f.jpg)

**Comment by aribudin** [comment, marked as answer]
I think you forgot `data-bs-toggle="dropdown"`

**Reply by LiviuSosu** [reply]
Thanks! I'm not a fron-end dev, I'm just exploring stuff, but it seems to me this [documentation ](https://getbootstrap.com/docs/4.0/components/dropdowns/) has issues at the moment

**Comment by bidyut18** [comment]
I downgraded my version to 4.0 and it's working fine. Bootstrap changed a
lot in V5. Naming conventions are different, I think that was the reason
data toggle was not mentioned. I'm new to bootstrap and still need to learn
a lot. Thank you for your response.

**Comment by aribudin** [comment]
You're welcome

good luck
